spyder/utils/color_system.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+# Color scales used for the default dark palette
+class Green:
+    B0 = '#000000'
+    B10 = '#064738'
+    B20 = '#055C49'
+    B30 = '#007A5E'
+    B40 = '#008760'
+    B50 = '#019D70'
+    B60 = '#02BA85'
+    B70 = '#20C997'
+    B80 = '#44DEB0'
+    B90 = '#3BEBB7'
+    B100 = '#88F2D3'
+    B110 = '#B0F5E1'
+    B120 = '#D1FBEE'
+    B130 = '#E4FFF7'
+    B140 = '#F5FFFD'
+    B150 = '#FFFFFF'
+
+
+class Red:
+    B0 = '#000000'
+    B10 = '#470606'
+    B20 = '#760B0B'
+    B30 = '#AF0F0F'
+    B40 = '#D4140B'
+    B50 = '#DE321F'
+    B60 = '#E24232'
+    B70 = '#E74C3C'
+    B80 = '#F66657'
+    B90 = '#F88478'
+    B100 = '#FFACA4'
+    B110 = '#FFC3BD'
+    B120 = '#FEDDDA'
+    B130 = '#FFEEEE'
+    B140 = '#FFF5F5'
+    B150 = '##FFFFFF'
+
+
+class Orange:
+    B0 = '#000000'
+    B10 = '#471D06'
+    B20 = '#692907'
+    B30 = '#AB3E00'
+    B40 = '#CE4B01'
+    B50 = '#E05E15'
+    B60 = '#E57004'
+    B70 = '#F37E12'
+    B80 = '#FF993B'
+    B90 = '#FFB950'
+    B100 = '#FFCF84'
+    B110 = '#FFDDA7'
+    B120 = '#FFEACA'
+    B130 = '#FFF3E2'
+    B140 = '#FFFBF5'
+    B150 = '##FFFFFF'
+
+
+class GroupDark:
+    """ Group colors for dark mode
+    Does not start with B0 because it doesn't use black """
```

Comment:
```suggestion
    """
    Group colors for the dark palette.

    It does not start with B0 because it doesn't use black.
    """
```

Just a bit of formatting to conform to the way we write docstring throughout our codebase.